src/v1/sources/shopify/pixelTransform.js

```diff
@@ -21,6 +25,32 @@ const NO_OPERATION_SUCCESS = {
   statusCode: 200,
 };
 
+const handleCartTokenRedisOperations = async (inputEvent, clientId) => {
+  try {
+    const cartTokenLocation = eventToCartTokenLocationMapping[inputEvent.name];
+    if (!cartTokenLocation) {
+      logger.info(`Cart token location not found for event: ${inputEvent.name}`);
+      return;
+    }
+    const cartToken = _.get(inputEvent, cartTokenLocation);
+    if (!cartToken) {
+      logger.info(`Cart token not found in input event: ${inputEvent.name}`);
```

Comment:
Instead of log, have metric to capture missing cart tokens

Reply:
added various metrics, removed info logs, retained the error log.